<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapOptionsScreen</class>
 <widget class="QWidget" name="MapOptionsScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #CCBCAD;</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>561</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Map mode:</string>
   </property>
   <widget class="QRadioButton" name="dynamicButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>dynamic</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="vectorButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>vector</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>793</x>
     <y>220</y>
     <width>88</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="latEdit">
   <property name="geometry">
    <rect>
     <x>672</x>
     <y>130</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lonEdit">
   <property name="geometry">
    <rect>
     <x>672</x>
     <y>160</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>130</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>lat</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>160</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>lon</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>100</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Coordinates center:</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>561</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="distance">
      <property name="title">
       <string>Distance radius(km):</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>561</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="cb_100">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>100</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_070">
          <property name="text">
           <string>70</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_050">
          <property name="text">
           <string>50</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_035">
          <property name="text">
           <string>35</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_025">
          <property name="text">
           <string>25</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_020">
          <property name="text">
           <string>20</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_005">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_002">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_001">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>871</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
